Modules/Services/Sources/Models/Block/BlockContent/Content/DataView/DataviewViewType.swift

```diff
@@ -38,6 +41,7 @@ public enum DataviewViewType: Hashable, CaseIterable {
         case .gallery: return "Gallery"
         case .kanban: return "Board"
         case .calendar: return "Calendar"
+        case .graph: return "Graph"
```

Comment:
Forgotten about  converter from DataviewTypeEnum to DataviewViewType on 57 line?

Reply:
You are right, thanks!